src/components/media/MediaFigure.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from 'react';
+import classnames from 'classnames';
+import './Media.scss';
+
+type Props = {
+    /** Component to use as outermost element, e.g., 'div' */
+    as: React.ElementType,
+    /** Child elements */
+    children: React.Node,
+    /** Additional class names */
+    className?: string,
+};
+
+function MediaFigure({ as: Wrapper, className, children, ...rest }: Props) {
+    return (
+        <Wrapper className={classnames('bdl-Media-figure', className)} {...rest}>
+            {children}
+        </Wrapper>
+    );
+}
+
+MediaFigure.defaultProps = {
```

Comment:
Not a big deal, but doing so I believe prevents uglifyJS from cleaning up MediaFigure. Since MediaFigure.defaultProps counts as a usage as such it cannot determine if MediaFigure was actually referenced.

Reply:
I'll see if using default values for the function shows up in styleguidist. I'd prefer that anyways.

Reply:
Oh it was because of this: https://github.com/facebook/flow/issues/5461

Reply:
I can either use `defaultProps` or use this syntax that doesn't report defaultValues to Styleguidist

```js
const MediaFigure = (props: Props) => {
    const { as: Wrapper = 'figure', className, children, ...rest } = props;
```

Reply:
Its fine I think to start using defaultProps for functional components. I am thinking we should make it a norm (so that flow is happy) and eat the theoretical minor bundle size increase. Maybe we can chat about it in #frontend